reg_tests/global_cycle/driver.hera.sh

```diff
@@ -29,7 +29,7 @@ module list
 
 WORK_DIR="${WORK_DIR:-/scratch2/NCEPDEV/stmp1/$LOGNAME}"
 
-PROJECT_CODE="${PROJECT_CODE:-fv3-cpu}"
+PROJECT_CODE="${PROJECT_CODE:-da-cpu}"
```

Comment:
Should this have been changed?

Reply:
Thanks for pointing this out. I have now reverted the project code back to the defaulted fv3-cpu.